transaction/core/src/onetime_keys.rs

```diff
@@ -18,6 +18,17 @@ use rand_core::{CryptoRng, RngCore};
 
 const G: RistrettoPoint = RISTRETTO_BASEPOINT_POINT;
 
+/// Generate a tx pubkey for a subaddress transaction
+pub fn compute_tx_pubkey(
```

Comment:
This just moved up in the file. I'm trying to organize the file in the order that the protocol uses the functions (the sender creates keys for the output, the recipient identifies an owned output, the recipient recovers the onetime private key for the output)